packages/animations/example/lib/container_transition.dart

```diff
@@ -48,20 +48,86 @@ class OpenContainerTransformDemo extends StatefulWidget {
 
 class _OpenContainerTransformDemoState
     extends State<OpenContainerTransformDemo> {
+  ContainerTransitionType transitionType = ContainerTransitionType.fade;
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(title: const Text('Container transform')),
+      appBar: AppBar(
+        title: const Text('Container transform'),
+        actions: <Widget>[
+          IconButton(
+            icon: Icon(Icons.settings),
+            onPressed: () {
```

Comment:
Maybe factor the onPressed handler out in its own method to make this build method shorter and easier to read?